pkg/apis/componentconfig/types.go

```diff
@@ -230,7 +230,7 @@ type KubeletConfiguration struct {
 	// be rejected.
 	LowDiskSpaceThresholdMB int `json:"lowDiskSpaceThresholdMB"`
 	// How frequently to calculate and cache volume disk usage for all pods
-	VolumeStatsAggPeriod unversioned.Duration `json:volumeStatsAggPeriod`
+	VolumeStatsAggPeriod unversioned.Duration `json:"volumeStatsAggPeriod"`
```

Comment:
@bgrant0607 This is what @goltermann is working torwards... as soon as we get the current codebase go vet clean, he will turn it on by default.